barman/cli.py

```diff
@@ -534,6 +543,8 @@ def backup(args):
             server.config.basebackup_retry_sleep = args.retry_sleep
         if args.retry_times is not None:
             server.config.basebackup_retry_times = args.retry_times
+        if args.keepalive_interval is not None:
```

Comment:
As mentioned by @martinmarques default should be 60, and here we should check not for none but for 0 value, I think. need to do a bit of research as argparse sometimes is confusing. let's discuss @gustabowill @barthisrael

Reply:
The arg parsed by `argparse` contains the `default` value when the user specifies nothing. If no `default` is specified, it brings `None`.

```python
import argparse

# Create the parser
parser = argparse.ArgumentParser(description="Example script with two optional integer arguments.")

# Add optional arguments
parser.add_argument(
    '--first',
    type=int,
    default=60,
    help='The first integer argument (default: 60).'
)

parser.add_argument(
    '--second',
    type=int,
    help='The second integer argument (no default value).'
)

# Parse the arguments
args = parser.parse_args()

# Access the arguments
print(f"First argument: {args.first}")
print(f"Second argument: {args.second}")
```

->

```console
$ python test.py
First argument: 60
Second argument: None
```